bundle/manifests/nvidia-network-operator.clusterserviceversion.yaml

```diff
@@ -58,11 +58,21 @@ spec:
     - kind: NicClusterPolicy
       name: nicclusterpolicies.mellanox.com
       version: v1alpha1
-  description: |
-    ## NVIDIA Network Operator
-    The NVIDIA Network Operator simplifies the provisioning and management of NVIDIA networking resources  in a Kubernetes cluster. The operator automatically installs the required host networking software - bringing together all the needed components to provide high-speed network connectivity. These components include the NVIDIA networking driver, Kubernetes device plugin, CNI plugins, IP address management (IPAM) plugin and others.
-    The NVIDIA Network Operator works in conjunction with the NVIDIA GPU Operator to deliver high-throughput, low-latency networking for scale-out, GPU computing clusters.
-    The Network Operator is open-source. For more information on contributions and release artifacts, see the [GitHub repo](https://github.com/Mellanox/network-operator/)
+  description: "## NVIDIA Network Operator\nThe NVIDIA Network Operator simplifies
```

Comment:
how would you update this ?

isnt it a hassle to add `\n` explicitly ?

using `: |` and just copy pasting what we got from README seems simpler to me.
although further updates are not expected to be large here so its not a big issue

Reply:
or is that auto generated from README and my comment is irrelevant :) ?

Reply:
I fixed that. It is taken from the template file :
`config/manifests/bases/nvidia-network-operator.clusterserviceversion.yaml`